packages/vaex-core/vaex/dataframe.py

```diff
@@ -4132,14 +4132,22 @@ def get_column_names(self, virtual=True, strings=True, hidden=False, regex=None)
         ['x', 'x2', 'y', 's']
         >>> df.get_column_names(regex='x.*')
         ['x', 'x2']
+        >>> df.get_column_names(regex='x.*')
+        ['x', 'x2']
+        >>> df.get_column_names(dtypes=[str])
+        ['s']
 
         :param virtual: If False, skip virtual columns
         :param hidden: If False, skip hidden columns
         :param strings: If False, skip string columns
         :param regex: Only return column names matching the (optional) regular expression
+        :param dtypes: A list of dtypes to consider
         :param alias: Return the alias (True) or internal name (False).
         :rtype: list of str
         """
+        if isinstance(dtypes, type):
+            dtypes = [dtypes]
```

Comment:
In almost all places in vaex we use singular form, and a list is optional. in this case 'dtype' would be more consistent. Agreed @JovanVeljanoski ?

Reply:
Agreed

Reply:
done